x-pack/plugins/endpoint/public/applications/endpoint/view/policy/policy_details.test.tsx

```diff
@@ -101,7 +101,8 @@ describe('Policy Details', () => {
         'EuiPageHeaderSection[data-test-subj="pageViewHeaderLeft"] EuiButtonEmpty'
       );
       expect(history.location.pathname).toEqual('/policy/1');
-      backToListButton.simulate('click');
+      backToListButton.simulate('click', { button: 0 });
+      await sleep();
```

Comment:
is this `sleep` necessary?

Reply:
Seems like a mistake/typo on my part. Will remove it. Thanks